event_track_assistant/models/event.py

```diff
@@ -347,6 +352,35 @@ def _update_presence_duration(self, hours, state=False, notes=False):
             vals['notes'] = notes
         self.write(vals)
 
+    def count_absences_create_claim(self):
+        claim_obj = self.env['crm.claim']
+        for presence in self:
+            presences = presence.event.mapped(
+                'track_ids.presences').filtered(
+                lambda x: x.partner == presence.partner and
+                x.session_date_without_hour <
+                presence.session_date_without_hour)
+            if (len(presences) >= 2 and
```

Comment:
if (len(presences) >= 2 and not presences[-2:].filtered(lambda x: x.state != 'absent')):